src/App.jsx

```diff
@@ -1,19 +1,24 @@
 import React, { Component } from 'react'
 import MenuItemList from './components/MenuItemList'
+import ViewOrder from './components/ViewOrder'
 import Registration from './components/Registration'
 import { Container, Header, Menu } from 'semantic-ui-react' 
 import LogIn from './components/LogIn'
 
 class App extends Component {
   state = {
-    category: "starters",
+    tab: "starters",
```

Comment:
why the change to tab from category?

Reply:
We added View Order button to navigation menu next. So now we had to change naming to something more general that categories, to make sense.

Reply:
Thanks for the explanation :)